.github/workflows/release_build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build release & deploy
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Resqu version has only 3 numbers, not 4.

Reply:
changed